src/printer.js

```diff
@@ -785,33 +777,52 @@ function genericPrintNoParens(path, options, print, args) {
             rightBrace
           ])
         );
-      } else {
-        return group(
```

Comment:
This looks like a scary PR but I just don't put a `group(` when it's going to be inlined.